src/module/config.service.ts

```diff
@@ -20,6 +20,13 @@ export type CustomHelper = {
   [key: string]: (...args: any[]) => any;
 };
 
+
+export interface ConfigOptions {
+  modifyConfigName?: (name: string) => string
+}
+
+export type Options = DotenvOptions & ConfigOptions;
```

Comment:
Consider extending `DotenvOptions` instead of using type aliase and `&` operator. 
As it's less readable and understandable.